sdks/python/src/opik/evaluation/metrics/llm_judges/answer_relevance/metric.py

```diff
@@ -1,6 +1,7 @@
 import json
 import logging
 from typing import Any, List, Optional, Union
+from pydantic import BaseModel
```

Comment:
please use the following import style (unless it's about importing generic types for type hints).
```python
import pydantic
class A(pydantic.BaseModel):
    pass
```
It may seem insignificant but it helps keeping the code easier to read and maintain.